xparser/src/tokens.rs

```diff
@@ -16,8 +16,8 @@ use super::{ParseInput, ParseResult};
 
 /// Reserved Keywords by jinko
 const RESERVED_KEYWORDS: &[&str] = &[
-    "func", "test", "mock", "type", "ext", "for", "while", "loop", "mut", "true", "false", "incl",
-    "as", "return", "where",
+    "func", "test", "mock", "type", "ext", "for", "while", "loop", "mut", "incl", "as", "return",
+    "where",
```

Comment:
Unsure if this is a good change or not - we'll have to see once we get to typechecking